network-protection/network-protection-impl/src/main/java/com/duckduckgo/networkprotection/impl/waitlist/NetworkProtectionWaitlistImpl.kt

```diff
@@ -141,3 +147,36 @@ class NetPRemoteFeatureWrapper @Inject constructor(
         return netPRemoteFeature.self().isEnabled() && netPRemoteFeature.waitlist().isEnabled()
     }
 }
+
+internal interface DeviceTimezoneProvider {
+    fun getTimeZone(): TimeZone
+}
+
+@Retention(AnnotationRetention.BINARY)
+@Qualifier
+private annotation class InternalApi
+
+// visible for testing
+internal typealias USTimeZoneChecker = () -> Boolean
+
+@Module
+@ContributesTo(AppScope::class)
+class TimezoneProviderModule {
+    @Provides
+    @InternalApi
+    fun provideUsTimezoneChecker(): USTimeZoneChecker {
+        return {
+            val defaultTimeZone = TimeZone.getDefault()
+            val usTimeZoneIDs = arrayOf(
+                "America/New_York",
```

Comment:
Any reason why we are not including: Adak, Metlakatla and Phoenix here? I see that they also belong to US timezone.

Reply:
No, I may be missing more timezones, will include them

Reply:
Can also check if id starts with “America/*” but we will miss out on Honolulu which is in reality quite far from the US region.